dkc/core/management/commands/mock_db.py

```diff
@@ -23,12 +23,12 @@ def _populate_subtree(
 
 
 @click.command()
-@click.argument('depth', type=click.INT)
-@click.argument('branching', type=click.INT)
-@click.argument('files', type=click.INT, default=0)
-@click.argument('user_id', type=click.INT, default=None)
-def command(depth: int, branching: int, files: int, user_id: int):
-    user = User.objects.get(id=user_id) if user_id else User.objects.first()
+@click.argument('depth', type=click.INT, default=3)
+@click.argument('branching', type=click.INT, default=3)
+@click.argument('files', type=click.INT, default=3)
+@click.option('--user-id', type=click.INT)
+def command(depth: int, branching: int, files: int, user_id: typing.Optional[int]):
+    user = User.objects.get(id=user_id) if user_id is not None else UserFactory()
```

Comment:
flip the condition to remove the `not` please

Reply:
Done.